tgui/src/interfaces/computer_fabricator.ract

```diff
@@ -29,16 +29,7 @@
 			<td><ui-button action='hw_netcard' params='{"netcard" : "0"}' state='{{data.hw_netcard == 0 ? "selected" : null}}'>None</ui-button>
 			<td><ui-button action='hw_netcard' params='{"netcard" : "1"}' state='{{data.hw_netcard == 1 ? "selected" : null}}'>Standard</ui-button>
 			<td><ui-button action='hw_netcard' params='{"netcard" : "2"}' state='{{data.hw_netcard == 2 ? "selected" : null}}'>Advanced</ui-button>
-		{{#if data.devtype != 2}} <!-- No tablets -->
```

Comment:
I just forgot about the comment probably.
Most of this is copied from my closed pr, on that one I spent ages trying to get the table to not break and look horrible.